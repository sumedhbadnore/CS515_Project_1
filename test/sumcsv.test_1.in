"Countries",Population,GDP
China,1409,15
India,1366,3
United States,331,22
Indonesia,276,1.3
Pakistan,225,0.3
Brazil,213,1.4
Russia,145,0.5
Mexico,130,1.2
