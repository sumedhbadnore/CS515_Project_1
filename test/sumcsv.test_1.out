Sum Of Column 'Population': 4095.0
Sum Of Column 'GDP': 44.699999999999996